pkg/operator/options/injectable.go

```diff
@@ -0,0 +1,41 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"context"
+	"flag"
+)
+
+// Injectable defines a set of flag based options to be parsed and injected
+// into Karpenter's contexts
+// type Injectable interface {
+// 	// Inject is used to inject the Options into the context and set up flags.
```

Comment:
If we have a set of comments like this, shouldn't these comments be directly over the function rather than at the top like this. General go convention is that you put your function comment directly above the function and the struct comment directly above the struct

Reply:
That's actually the old Injectable struct that I commented out and forgot about. I'll move the comments.